src/utils/people.ts

```diff
@@ -128,3 +128,64 @@ export async function claimOnLiquid({
     throw e
   }
 }
+
+export async function createBadge({ host, icon, amount, name, owner_pubkey }) {
+  try {
+    const token = await genSignedTimestamp(owner_pubkey)
+    let protocol = 'https'
+    // if (config.tribes_insecure) protocol = 'http'
```

Comment:
is this meant to be commented out?

Reply:
Ops!!!.....

Nope, would push a fix for that now